internal/publish/publish.go

```diff
@@ -65,21 +65,25 @@ type publishHandler struct {
 	authorizedAppProvider authorizedapp.Provider
 }
 
-// There is a target normalized latency for this function. This is to help prevent
-// clients from being able to distinguish from successful or errored requests.
-func (h *publishHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+type response struct {
+	status          int
+	message         string
+	metric          string
+	count           int // metricCount
+	errorInNonDebug bool
```

Comment:
naming nit: could put this in the affirmative as "errorInProd" to avoid double negatives.

Reply:
good call.